clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -861,7 +861,7 @@ void ContinuationIndenter::addTokenOnCurrentLine(LineState &State, bool DryRun,
       //  or
       //  caaaaaaaaaaaaaaaaaaaaal(
       //       new SomethingElseeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee());
-      !IsSimpleFunction(Current)) {
+      Current.isNot(tok::comment) && !IsSimpleFunction(Current)) {
```

Comment:
> would it make sense to introduce this into `IsSimpleFunction` ? similar to handling of `new` keyword in there?

I did that at first but thought it would be confusing to say that a comment is a simple function (or function call).